src/LanguageServer/Impl/Indexing/IndexManager.cs

```diff
@@ -58,36 +59,23 @@ internal class IndexManager : IIndexManager {
             _addRootTcs = new TaskCompletionSource<bool>();
```

Comment:
Minor style thing, but all of the assignments you're making to the `readonly` members here can be moved up above, as they only depend on static values.